02-fridge-fleet-monitor/README.md

```diff
@@ -17,71 +12,110 @@ To run this project yourself you'll need to:
+## Testing
 
-![Sparrow door switches aligned together when door is closed](images/readme-door-switches-closed.jpg)
+Ensure your Notecarrier is powered with a battery or via a USB power supply.
 
-Notice multiple Command Strips raise the connected contact switch sensor on the fridge so they're close to level with the switch on the door's edge.
+Connect the Raspberry Pi Pico to your computer with a USB Cable, ensuring you have no power cable between the Notecarrier and the Pico to prevent damage.
 
-![Sparrow door switches separated when door is opened](images/readme-door-switches-open.jpg)
+Open a Serial Terminal Emulator to view the log output from the Pico Code
+You should see output similar to below:
 
-Here's how it looks opened. The rubber seal of the door does not interfere with or impede the contact switches from correctly reporting to the Sparrow node inside the fridge at all.
+![example output from serial console of Pico](images/pico-serial-output.png)
 
-Now that the harware is installed, the last task left is to set up the web app.
-## Web Application
+Open the fridge door.  In the console you should see the message 'Door Wakeup. Door Value: True' and 'Door Open'.  In addition, you should see an event in the Blues Notehub with a filename of `door.qo` with the body of `{'open':true}`
 
-The Fridge Fleet Monitor's web application is based on the [Sparrow Reference Web App](https://github.com/blues/sparrow-reference-web-app). 
+Close the fridge door. In the console you should see the message 'Door Closed'.  In addition, you should see an event in the Blues Notehub with a filename of `door.qo` with the body of `{'closed':true}`
 
-Refer to the [`README.md`](https://github.com/blues/app-accelerators/tree/main/02-fridge-fleet-monitor/web-app/README.md) file in this project's [`web-app`](https://github.com/blues/app-accelerators/tree/main/02-fridge-fleet-monitor/web-app) folder for full setup instructions.
+Aditionally you should also see an event in Notehub with the filename of `readings.qo` and the body similar to `{"humidity":48,"temperature":22}`. This is the relative humidity percentage and the temperature in celcius.  This event should be sent by default every 60 minutes.
 
-If all goes well, you'll end up with a Fridge Fleet Monitor dashboard similar to these.
+## Further Development
 
-![Fridge Fleet Monitor web app dashboard](images/readme-refrigerator-fleet-monitor-dashboard.png)
+If you wish to develop this project further, here are some suggestions.
 
-_Fridge Fleet Monitor main dashboard._
+Firstly, the DHT11 sensor is cheap and accessable, but not nescarcellary the most accurate.  Consider replacing it with another, for example the BME280 which connects via I2C, so will share the same connections as the Notecarrier, and there is a [CircutPython Library available as well](https://docs.circuitpython.org/projects/bme280/en/latest/)
```

Comment:
```suggestion
Firstly, the DHT11 sensor is cheap and accessible, but not necessarily the most accurate.  Consider replacing it the BME280, which also connects via I2C, so it will share the same connections as the Notecarrier. There is a [CircutPython Library available for the BME280 as well](https://docs.circuitpython.org/projects/bme280/en/latest/)
```